web/packages/teleport/src/services/websession/websession.ts

```diff
@@ -36,11 +36,20 @@ let sesstionCheckerTimerId = null;
 
 const session = {
   logout(rememberLocation = false) {
-    api.delete(cfg.api.webSessionPath).finally(() => {
-      history.goToLogin(rememberLocation);
+    api.delete(cfg.api.webSessionPath).then(response => {
+      if (response.redirect) {
+        this.clear();
+        window.open(response.redirect, '_self');
```

Comment:
> I couldn't get http.Redirect to work because I would get an unsafe attempt to load URL

From what I see, redirections can't be done from DELETE requests, so this is probably the reason.

I'm only wondering if returning the URL from the server, which is then blindly opened by the web UI isn't less secure 🤔 But OTOH we trust the backend side, and if frontend is compromised it can open anything it wants anyway.

Btw, I'd maybe call this field `samlSloUrl` instead of `redirect`.